packages/apps/spaces/app/organizations/[id]/components/Tabs/shared/FormSocialInput/FormSocialInput.tsx

```diff
@@ -175,7 +176,7 @@ export const FormSocialInput = ({
 
   return (
     <>
-      {(values as Value[])?.map(({ id, url }, index) => (
```

Comment:
Let's name types as entities they are describing, eg `SocialInputValue`. That makes the code easier to read and understand what is happening especially after a while